benchmarks/prova.js.rb

```diff
@@ -1,13 +0,0 @@
-# 'asdf' =~ /(a)./
-#
-# p $'
-# p $1
-# p $2
-# `console.log(2, $opal.gvars['1'])`
-# b = $1
-# `console.log(1, b)`
-# nil
-
-puts [*1..3].to_s
-puts [*(1..3)].to_s
-puts [*Object.new].to_s
```

Comment:
Hehe, no problem @elia, it wasn't doing any harm and actually helped me get by bearings.